pucauto.py

```diff
@@ -21,7 +22,7 @@
 
 def print_pucauto():
     """Print logo and version number."""
-
+    # avoid writing the banner to anyplace but the console...
     print("""
```

Comment:
I didn't want to clutter up the logs with a big ascii art diagram. This just `print`s instead.